### PR TITLE
Add and fix Util doc

### LINKED ISSUE #48: fix weird whitespace
why the indent looks weird

_Originally posted by @ntut-rick in https://github.com/ntut-open-source-club/practical-tools-for-simple-design/pull/46#discussion_r1342685576_

### LINKED ISSUE #72: Document `namspace Util`
- [x] [BGM.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/BGM.hpp)
- [x] [Docs.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/Docs.hpp)
- [ ] [GameObject.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/GameObject.hpp) #90
- [ ] [Image.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/Image.hpp) #90
- [x] [Input.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/Input.hpp)
- [x] [Keycode.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/Keycode.hpp)
- [x] [LoadTextFile.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/LoadTextFile.hpp)
- [ ] [Logger.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/Logger.hpp) #90
- [x] [SFX.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/SFX.hpp)
- [ ] [Text.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/Text.hpp) #90
- [ ] [Time.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/Time.hpp) #90
- [ ] [Transform.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/Transform.hpp) #90
- [ ] [TransformUtils.hpp](https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/main/include/Util/TransformUtils.hpp) #90

### PR DESCRIPTION
**Pull request by Dragonfly911117**
Fixes #72, Fixes #48

### CODE REVIEW

File: include/Util/Transform.hpp [hunk 1]
```diff
@@ -5,11 +5,40 @@
...
+     * This property represents the scale of the transformation in 2D space.
+     * It is represented as a 2D vector.
+     */
     glm::vec2 scale = {1, 1};
 };
+
 } // namespace Util
 
-#endif /* UTIL_TRANSFORM_HPP */
+#endif /* UTIL_TRANSFORM_HPP */
```
[the remark this answers is not in the document]
Comment (NOOBDY) [reply include/Util/Transform.hpp:44]: i think this is caused by EOF linebreak

File: include/Util/Transform.hpp [hunk 2]
```diff
@@ -5,11 +5,41 @@
...
+     * space. It is represented as a 2D vector.
+     */
     glm::vec2 translation = {0, 0};
+
+    /**
+     * @brief The rotation of the transformation.
+     *
+     * This property represents the rotation of the transformation in 2D space.
+     * It is represented as a float.
+     * @note The unit of rotataion is in radians not degrees.
```
Comment (NOOBDY) [inline include/Util/Transform.hpp:30]:
```suggestion
     * @note The unit of rotation is in radians not degrees.
```

File: include/Util/Time.hpp [hunk 3]
```diff
@@ -4,26 +4,60 @@
...
+     *
+     * @return The delta time between frames in seconds.
      */
     static double GetDeltaTime() { return s_DeltaTime; }
 
+    /**
+     * @brief Update the time.
+     *
+     * This function updates the current time and the delta time.
+     * It is called by Cord::Context::Update(). The user is not recommanded to
```
Comment (NOOBDY) [inline include/Util/Time.hpp:33]:
```suggestion
     * It is called by Cord::Context::Update(). The user is not recommended to
```

**Review by NOOBDY — CHANGES_REQUESTED** [review]
small typos

**Review by ntut-rick — CHANGES_REQUESTED** [review]

**Review by NOOBDY — APPROVED** [review]

**Review by NOOBDY — APPROVED** [review]

**Review by ntut-rick — APPROVED** [review]

**Pull request merged**